sass/layout/_headers.scss

```diff
@@ -59,20 +75,37 @@
     }
 
```

Comment:
Can we change the padding on the channel-header__trigger?
```
.channel-header .channel-header__trigger {
    padding: 4px 6px;
}
```

And update the type style on the channel-header heading

```
.channel-header .heading {
    font-size: 14px;
    line-height: 16px;
}
```

And change the following on the dropdown icon
```
.header-dropdown__icon {
    font-size: 16px;
    line-height: 16px;
    margin: 0 0 0 4px;
}
```